docs/learn/dapp-developer/mint-lsp7-token.md

```diff
@@ -15,40 +15,84 @@ In this guide you will mint some [LSP7 Digital Asset](../../standards/tokens/LSP
 
 The following code snippets require the installation of the following libraries:
 
-- [`ethers.js`](https://github.com/ethers-io/ethers.js/)
+- [`ethers.js`](https://github.com/ethers-io/ethers.js/) or [`web3.js`](https://www.npmjs.com/package/web3)
 - [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts/)
 
+<Tabs groupId="web3-lib">
+  <TabItem value="web3js" label="web3.js">
+
+```shell
+npm install web3 @lukso/lsp-smart-contracts
+```
+
+  </TabItem>
+  <TabItem value="ethersjs" label="ethers.js">
+
 ```shell
 npm install ethers @lukso/lsp-smart-contracts
 ```
 
+  </TabItem>
+</Tabs>
+
 ### Imports and constants
 
-At this point, the `LPS7Mintable` contract is being prepared for the following interaction. You construct an instance of a contract, using its _ABI_ and the _contract address_.
+At this point, the [`LSP7Mintable`](../../contracts/contracts/LSP7DigitalAsset/presets/LSP7Mintable.md) contract is being prepared for the following interaction. You construct an instance of a contract, using its _ABI_ and the _contract address_.
 
-```javascript
+<Tabs groupId="web3-lib">
+  <TabItem value="web3js" label="web3.js">
+
+```javascript title="web3.js"
 import LSP7Mintable from '@lukso/lsp-smart-contracts/artifacts/LSP7Mintable.json';
-import { ethers } from 'ethers';
+import Web3 from "web3";
 
-const privateKey = '0x...';
 const myTokenAddress = '0x...';
-const universalProfileAddress = '0x...';
 
-const provider = new ethers.JsonRpcProvider('https://rpc.testnet.lukso.network');
+const web3 = new Web3(window.lukso);
 
-// setup signer
-const signer = new ethers.Wallet(privateKey as string, provider);
-console.log('🔑 EOA: ', signer.address);
+await web3.eth.requestAccounts();
+const accounts = await web3.eth.getAccounts();
 ```
 
+  </TabItem>
+  <TabItem value="ethersjs" label="ethers.js">
+
+```javascript title="ethers.js"
+import LSP7Mintable from '@lukso/lsp-smart-contracts/artifacts/LSP7Mintable.json';
+import { ethers } from 'ethers';
+
+const myTokenAddress = '0x...';
+
+await ethers.provider.send("eth_requestAccounts", []);
+const signer = ethers.provider.getSigner();
+```
+
+  </TabItem>
+</Tabs>
+
+
 ### Instantiate the contracts
 
-After defining the core parameters of the [`LPS7 Mintable`](../../contracts/contracts/LSP7DigitalAsset/presets/LSP7Mintable.md) contract, you are able to create an instance using its _ABI_ and the _contract address_.
+After defining the core parameters of the [`LSP7 Mintable`](../../contracts/contracts/LSP7DigitalAsset/presets/LSP7Mintable.md) contract, you are able to create an instance using its _ABI_ and the _contract address_.
```

Comment:
`LSP7 Mintable` (with a space as per this line)
or `LSP7Mintable` (with no space as per line 40) ?

id say lets use the same everywhere

Reply:
As we wrote in `CONTRIBUTING.md`

- With space if the standard is referenced (better readability)
- Without space if the Solidity Contract is referenced

I will fix build issues and apply this suggestion